pkgs/development/tools/goresym/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "goresym";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "mandiant";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-XDy6qALE2xuOVQHuuvU/gOyTRZ/6CWpyjN3i7VmznoI=";
+  };
+
+  subPackages = [ "." ];
+
+  vendorSha256 = "sha256-dnQ/tP4RS6WkACobfW7jTTJSHbLrdlZDy1fmO65743Q=";
+
+  doCheck = false;
```

Comment:
Can you add a comment explaining why the tests must be disabled?

Reply:
Fixed. Tests did not need to be disabled.